content/student/relax.md

```diff
@@ -0,0 +1,139 @@
+title: На досуге
+slug: relax
+category: student
+tags: [_nav]
+submenu_index: 8
+type: tag_by_slug_sidebar
+---
+
+<div class="relax-content">
```

Comment:
I would suggest moving all the HTML content into a new template. Remember to format the page properly!